sites/website/src/docs/community/engineering-process/issue-management.md

```diff
@@ -0,0 +1,120 @@
+
+*In general* we don't mind getting duplicate issues. It's easier for us to close duplicate issues than to discuss multiple root causes on a single issue! We may close your issue as a duplicate if we determine it has the same root cause as another. Don't worry! It's not a waste of our time!
+
+### Needs Information
+
+If a contributor reviews an issue and determines that more information is needed from the author, they will post a comment requesting that information and apply the `status:needs-information` label. This label indicates that the author needs to post a response in order for us to continue investigating the issue.
+
+If the author does not post a response within **7 days**, the issue may be closed. If the author responds within **7 days** after the issue is closed, the issue will be  re-opened. We recognize that you may not be able to respond immediately to our requests, we're happy to hear from you whenever you're able to provide the new information.
+
+### PR: Needs Author Input
+
+Similar to the `status:needs-information` process above, PRs also require author input from time to time. When a member of our team asks for some follow-up changes from the author, we mark these PRs with `status:needs-author-input` label. After doing so, we expect the author to respond within 14 days.
+If the author does not post a response or updates the PR within **14 days**, the issue may be closed. If the author responds within **7 days** after the issue is closed, the issue will be re-opened. We recognize that you may not be able to respond immediately to our requests, we're happy to hear from you whenever you're able to provide the new information.
+
+### Duplicate Issues
+
+If we determine that the issue is a duplicate of another, we will label it with the `closed:duplicate` label, link it to the duplicate, and close the issue.
+
+### Answered Questions
+
+If we determine that the issue is a question, we will label it with `community:question`. If we have posted an answer, we will label it with the `closed:done` label. The issue will be then be closed.
+
+### Locking Closed Issues
+
+After an issue has been closed and had no activity for **30 days** it may be locked. This is done in order to reduce confusion as to where to post new comments. If you are still encountering the problem reported in an issue, or have a related question or bug report, feel free to open a *new issue* and link to the original (now locked) issue!
+
+## Acknowledgements
+
+Many thanks to the .NET team for sharing their policies and practices upon which this document was based.
```

Comment:
Is there a link we can include for the .NET team?  Cross-linking with credible partners is great for SEO.

Reply:
I'll think I missed a few things. I'll check in and make sure this matches our other docs and is consistent throughout.